src/proxy-ui-api/src/main/resources/openapi-definition.yaml

```diff
@@ -4895,6 +4895,19 @@ components:
           items:
             type: string
             format: text
+        validation_errors:
+          description: object that contains validation errors bound to their respected fields - the object's key
```

Comment:
would mentioning "dictionary" help to make this structure more clear? I was a bit befuddled at first, not understanding how this results in `Map<String, List<String>> validationErrors` (granted, due to my lack of knowledge in openapi)

Reply:
Tried to be more verbose about this. Also added a link to OAS3 Dictionary docs right after the `additionalProperties` keyword